src/main/java/bio/terra/cli/serialization/userfacing/inputs/UpdateResourceParams.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.cli.serialization.userfacing.inputs;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+/**
+ * Parameters for updating a workspace resource. This class is not currently user-facing, but could
+ * be exposed as a command input format in the future. This class handles properties that are common
+ * to all resource types. Sub-classes include additional resource-type specific properties.
```

Comment:
fix comment, this doesn't use subclasses.

Reply:
left over from a previous iteration. fixed here and in the create version.